runtime/src/lib.rs

```diff
@@ -307,8 +278,10 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 500;
+    pub const ExistentialDeposit: u128 = 1 * MILLIUNIT;
```

Comment:
we used `DOLLARS` in our standalone chain https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR522,   why do they do it differently in the substrate-parachain-template than the substrate-node-template?
what is more ideal?

Reply:
I am not sure. I have asked here https://github.com/paritytech/subport/issues/246